in-cloud/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Meticulous - Report Diffs (In Cloud)"
+description: "Run automated Meticulous tests in Meticulous' cloud."
+inputs:
+  api-token:
+    description: "Meticulous API token"
+    required: true
+  github-token:
+    description: "GITHUB_TOKEN or a repo scoped PAT."
+    required: true
+    default: ${{ github.token }}
+  app-url:
+    description: |
+      The URL to execute the tests against. This URL should serve the code from the current commit (e.g. a localhost URL served up by a local server).
+    required: true
+
+outputs: {}
+runs:
+  using: node20
+  main: "../out/in-cloud.entrypoint.js"
```

Comment:
`run-all-tests-in-cloud` makes sense because it directly describes the action; but `report-diffs-action/in-cloud` isn't quite as smooth / sounds more like it's reporting diffs in the cloud rather than running tests in the cloud (but I guess it's actually doing both)

Another option would be `report-diffs-action/cloud-compute` -- maybe that sounds better, and more powerful, and makes it a little clearer what's going on ('where do you want to run your compute?')? What do you think? I could go either way.

Reply:
https://github.com/alwaysmeticulous/report-diffs-action/pull/405/commits/99de711b56cd3f1aa1e9f4a2158a60ddce8e3156 does that